spec/controllers/api_entreprise/authorization_requests_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe APIEntreprise::AuthorizationRequestsController do
```

Comment:
Honnêtement je pense que tu peux flip les tests de controller, ça va plus être relou qu'autre chose à maintenir alors qu'in fine ce qu'on veut tester c'est le comportement final, testé dans la feature spec.

Reply:
je les ai delete dans un des commits mais le suivant les as remis T_T.

Reply:
😅